src/main/resources/application.properties

```diff
@@ -1 +1,19 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://localhost:3306/----
+spring.datasource.username=-------
+spring.datasource.password=-------
 
+spring.profiles.active=dev
+
+spring.servlet.multipart.max-file-size=1024KB
+spring.servlet.multipart.max-request-size=1024KB
+
+spring.kafka.consumer.bootstrap-servers=localhost:9092
+spring.kafka.consumer.group-id=fastfoodapplication
+spring.kafka.consumer.auto-offset-reset=earliest
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+
+spring.kafka.producer.bootstrap-servers=localhost:9092
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
```

Comment:
There's a bit of new/unused functionality here.  Database and Kafka.  Only add these when they are required: the feature is complete.  Otherwise, it is an incomplete feature, which is clutter, which is technical debt.  In other words, don't bloat code for something that might be used in the future.